stdlib/2/SimpleHTTPServer.pyi

```diff
@@ -0,0 +1,16 @@
+# Stubs for SimpleHTTPServer (Python 2)
+
+from typing import Any, AnyStr, IO, Mapping, Optional
+import BaseHTTPServer
+from StringIO import StringIO
+
+class SimpleHTTPRequestHandler(BaseHTTPServer.BaseHTTPRequestHandler):
+    server_version = ...  # type: str
+    def do_GET(self) -> None: ...
+    def do_HEAD(self) -> None: ...
+    def send_head(self) -> Optional[IO[AnyStr]]: ...
+    def list_directory(self, path: str) -> Optional[StringIO]: ...
+    def translate_path(self, path: str) -> str: ...
+    def copyfile(self, source: IO[AnyStr], outputfile: IO[AnyStr]): ...
+    def guess_type(self, path: str) -> str: ...
```

Comment:
So `Union[str, unicode]` instead of `AnyStr` for `path`? Or does it not make any difference here?